ocr.py

```diff
@@ -44,3 +49,116 @@ def ocr(self, image_file, show_steps=False):
         features = self.feature_extractor.extract(image_file.image, segments)
         classes = self.classifier.classify(features)
         return classes, segments
+
+
+class SimpleOCR(object):
+    def __init__(self, segmentation="contour", extraction="simple", classification="knn",
```

Comment:
Well, I didn't really want to mess with your code as it would be merged back in here, but I think that most of the functions I wanted to add could indeed easily be integrated in the normal OCR class. This would avoid code duplication and probably make the code more transparent, so I think that's a really good idea. What functions would you like to integrate in the OCR class?